bat.go

```diff
@@ -231,70 +232,39 @@ func main() {
 		return
 	}
 
-	if runtime.GOOS != "windows" {
-		fi, err := os.Stdout.Stat()
-		if err != nil {
-			panic(err)
-		}
-		if fi.Mode()&os.ModeDevice == os.ModeDevice {
-			if printV == "A" || printV == "H" || printV == "B" {
-				dump := httpreq.DumpRequest()
-				if printV == "B" {
-					dps := strings.Split(string(dump), "\n")
-					for i, line := range dps {
-						if len(strings.Trim(line, "\r\n ")) == 0 {
-							dump = []byte(strings.Join(dps[i:], "\n"))
-							break
-						}
-					}
-				}
-				fmt.Println(ColorfulRequest(string(dump)))
-				fmt.Println("")
-			}
-			if printV == "A" || printV == "h" {
-				fmt.Println(Color(res.Proto, Magenta), Color(res.Status, Green))
-				for k, v := range res.Header {
-					fmt.Println(Color(k, Gray), ":", Color(strings.Join(v, " "), Cyan))
-				}
-				fmt.Println("")
-			}
-			if printV == "A" || printV == "b" {
-				body := formatResponseBody(res, httpreq, pretty)
-				fmt.Println(ColorfulResponse(body, res.Header.Get("Content-Type")))
-			}
-		} else {
-			body := formatResponseBody(res, httpreq, pretty)
-			_, err = os.Stdout.WriteString(body)
-			if err != nil {
-				log.Fatal(err)
-			}
-		}
-	} else {
-		if printV == "A" || printV == "H" || printV == "B" {
-			dump := httpreq.DumpRequest()
-			if printV == "B" {
-				dps := strings.Split(string(dump), "\n")
-				for i, line := range dps {
-					if len(strings.Trim(line, "\r\n ")) == 0 {
-						dump = []byte(strings.Join(dps[i:], "\n"))
-						break
-					}
+	fi, _ := os.Stdout.Stat()
```

Comment:
I can compile and run this under windows 8 with go1.4.2.

Screenshot:
![without-color](https://cloud.githubusercontent.com/assets/281635/7785324/0fa49a62-0194-11e5-9d63-879b3169f488.png)

After setting appropriate colors(ansi color codes between 30-39):
![with-color](https://cloud.githubusercontent.com/assets/281635/7785328/21a16e66-0194-11e5-9934-b2a1ff2d63c8.png)